src/security/Session.h

```diff
@@ -18,6 +18,31 @@
 #if USE_OPENSSL
 #if HAVE_OPENSSL_SSL_H
 #include <openssl/ssl.h>
+
+/// These are not included by openssl/ssl.h when deprecated API is disabled
+#include <openssl/opensslv.h>
+#include <openssl/bn.h>
+#include <openssl/rsa.h>
+#include <openssl/dh.h>
+#include <openssl/x509.h>
+
+/// Use new calls for API deprecated in OpenSSL 1.1.0
+/// Note that all of the calls in this next group were implemented at the same time
+/// as X509_STORE_CTX_set0_untrusted, in both OpenSSL 1.1.0, and LibreSSL 2.7.0.
```

Comment:
This statement is not completely true. At least LibreSSL 2.7.4 still contains SSLeay_version() symbols.